charts/netdata/Chart.yaml

```diff
@@ -14,15 +14,11 @@ sources:
 maintainers:
   - name: Ilya Mashchenko
     email: ilya@netdata.cloud
-  - name: Konstantinos Natsakis
-    email: knatsakis@netdata.cloud
-  - name: Mansour Behabadi
-    email: mansour@netdata.cloud
-  - name: Vladimir Ryumin
-    email: vryumin@gmail.com
-  - name: Karuppiah Natarajan
-    email: karuppiah7890@gmail.com
-  - name: Oleksii Kravchenko
-    email: kamikaze.ua@gmail.com
+  - name: Mateusz Bularz
+    email: mat@netdata.cloud
+  - name: Bill Kalaitzis
+    email: bill@netdata.cloud
+  - name: Witold Duranek
+    email: witold@netdata.cloud
```

Comment:
Probably a better way to put a team here, not individual people.

Reply:
Agreed. Perhaps we should create an internal alias specifically for those maintaining the Helm chart?

Reply:
I was thinking of just using the Cloud SRE team. I don't think that is so important we need a dedicated alias just for helmchart maintainers.

Reply:
Well yes, but it’s not _just_ the Cloud SRE team.

Reply:
Ok, maintainers is [optional](https://helm.sh/docs/topics/charts/#the-chartyaml-file). I think we can just remove the field. I think it is ok, maintained by Netdata.

Reply:
@witalisoft added Cloud SRE team, please approve